docs/Oracle Jobs/Job Types/cron.md

```diff
@@ -6,6 +6,7 @@ permalink: "docs/jobs/types/cron/"
 ---
 
 Executes a job on a schedule. Does not rely on any kind of external trigger.
+This means it's triggered based on some condition that the Chainlink node evaluates, and not triggered externally via a smart contract. Because of this, the node isn't paid in LINK tokens for processing the request like it does for API calls, because it's initiating a request itself as opposed to receiving a request (and payment) from on-chain.
```

Comment:
```suggestion
This it's triggered based on some condition that the Chainlink node evaluates, and not triggered externally via a smart contract. Because of this, the node isn't paid in LINK tokens for processing the request like it does for API calls, because it's initiating a request itself as opposed to receiving a request (and payment) from on-chain.
```